CMakeLists.txt

```diff
@@ -100,22 +100,29 @@ endif( )
 # BUILD_SHARED_LIBS is a cmake built-in; we make it an explicit option such that it shows in cmake-gui
 option( BUILD_SHARED_LIBS "Build hipBLAS as a shared library" ON )
 
-# Find CUDA if the user wants a CUDA version.
-option(USE_CUDA "Look for CUDA and use that as a backend if found" OFF)
-if (USE_CUDA)
-	find_package( CUDA REQUIRED )
+# Deprecated USE_CUDA option
+if(DEFINED USE_CUDA)
+  if(USE_CUDA)
+    set(ENV{HIP_PLATFORM} nvidia)
+    message(DEPRECATED "USE_CUDA is deprecated (use HIP_PLATFORM=nvidia)")
```

Comment:
Is it an env variable or cmake variable for HIP_PLATFORM ?  You set env but use variable so it is confusing.

Reply:
If the user wants to override the auto-detected platform, they can set the environment variable HIP_PLATFORM. Once find_package(hip) is called, it sets the variable HIP_PLATFORM to be used in cmake. I've added a link to the document Cory created in the description.

Reply:
So would -DHIP_PLATFORM=nvidia be the normal usage for cuda or env if you have both installed?

Reply:
If you're building with cmake I believe you'll still want to set the environment variable to force HIP to find the platform you want.